integration/Main.java

```diff
@@ -115,12 +115,16 @@ private static void initializeModules()
 
 		// train model <---> MBO
 		//for (TrainModel trainModel : trainModels)
-		//	mbo.registerTrain(trainModel.mboRadio());
+		//	mboCont.registerTrain(trainModel.id(), trainModel.mboRadio());
```

Comment:
This made me realize there's still a class variable `mbo` above that will have to be changed too. Not worth worrying about.

https://github.com/suffolk-northern/main/pull/28/files#diff-0987bc73c294ac5a077d006ab3be4dfcL37